LibTCPServer.emx

```diff
@@ -459,27 +446,20 @@ srv->start();
 			}&#xD;
 &#xD;
 			// TODO: Check so that dataType is compatible with foundSignalDescriptor->type&#xD;
-&#xD;
 	    	//foundSignalDescriptor->type->decode(data, &amp;decoder);&#xD;
 	    }&#xD;
 &#xD;
 	    // Is a port index specified?&#xD;
 	    int portIndex = -1;&#xD;
-	    try {&#xD;
-	    	Var var_portIndex = object->get(&quot;portIndex&quot;);&#xD;
-	    	if (!var_portIndex.isEmpty())&#xD;
-	    		portIndex = var_portIndex.convert&lt;int>();&#xD;
-	    }&#xD;
-	    catch (Poco::Exception e) {&#xD;
-	    	conn->replyStatus(ITCPConnection::Status::Error, &quot;Illegal portIndex: &quot; + e.message());&#xD;
-	    	return;&#xD;
-	    }&#xD;
-	    &#xD;
+		std::string portIndexString =  mp.find(&quot;portIndex&quot;)->second;&#xD;
+		if (!portIndexString.empty()) {&#xD;
+			portIndex = stoi(portIndexString);&#xD;
```

Comment:
Exception can be use for error handling with stoi() function but we have to avoid exception implementation. So I am planning to use stringstream class here.

Reply:
@rajneshkumarmalik Why are we avoiding exceptions now? We have been using them a lot before to catch errors from poco.

Reply:
In bugbash meeting Mattias told to avoid exceptions. BTW it's slow down the execution.